### ISSUE TITLE
E21: Cannot make changes, 'modifiable' is off

### ISSUE THREAD

**Issue by clauseggers** [mrtazz/simplenote.vim#93]
I’m running the latest version on TrueOS (BSD), and I keep getting this curious error when I start Simplenote: 
``
Error detected while processing /usr/home/claus/.vim/bundle/simplenote.vim/autoload/simplenote.vim:
line 22:
E21: Cannot make changes, 'modifiable' is off
"Simplenote" [New File]
Press ENTER or type command to continue
``

Any ideas on what is going wrong?

**Comment by atomicules** [comment]
No, but I have found another issue with the saveas functionality as a result of investigating this. Gah!

What steps are you doing? E.g:

1. Open vim
2. `:SimplenoteNew`
3. Type some text
4. `:w`

(The issue I've just found).

**Comment by clauseggers** [comment]
What I do is:
1. Open Vim
2. :SimplenoteList
… and that gives me the error as described above.

These are my `.vimrc` options:
```
source $HOME/.simplenotevimrc
let g:SimplenoteSortOrder = "pinned, modifydate"
let g:SimplenoteFiletype = "txt"
" let g:SimplenoteListHeight = 10
let g:SimplenoteSingleWindow = 1
map <F4> :SimplenoteList<CR>
nnoremap <Leader>nn :SimplenoteNew<CR>
```

**Comment by atomicules** [comment]
That is strange. I use SimplenoteList a lot and haven't seen any issues. I'll see if I can reproduce with your `.vimrc` options though.

**Comment by clauseggers** [comment]
It could very well be a TrueOS issue. It’s a new(ish) distro with cutting edge components.

**Comment by derteta** [comment]
FWIW, I see this error in my command-line `vim` (8.0, build from the git repo) but not in MacVim (7.4).

**Comment by atomicules** [comment]
Tried on MacVim 8.0 and don't see this problem. I'll have to go build a 8 version on NetBSD and check there.

**Comment by atomicules** [comment]
Drat! Just built 8.0.194 on NetBSD and can't reproduce there either. This is a strange one.

**Comment by mplorentz** [comment]
Same is occuring for me on Arch Linux, vim 8.0.

**Comment by verajosemanuel** [comment]
same error here . Using Arch
